storage.py

```diff
@@ -0,0 +1,43 @@
+from google.cloud import storage
+from db import Db
+
+def upload_cover_blob(bucket_name, source_file_name, destination_blob_path): 
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    destination_blob_name = f'''{destination_blob_path}/coverimg'''
+    blob = bucket.blob(destination_blob_name)
+    blob.upload_from_file(source_file_name)
+    # blob.upload_from_filename(source_file_name.filename)
+    blob.make_public()
+
+    print(
+        "File {} uploaded to {}.".format(
```

Comment:
ใช้ single quote แล้วเขียน format แบบนี้แทน
f'File {source_file_name.filename} updated to {destination_blob_name}.'